word-search.py

```diff
@@ -81,34 +80,91 @@ def print_grid(grid):
     # Bottom border
     print(" " * left_padding + "└" + "─┴" * (cols - 1) + "─┘")
 
-# Test only. Create and print the grid using the selected size
-grid = create_grid(gridsize)
-print_grid(grid)
-
 def loadWordLibrary():
     # read words from file and return as list for use in program
     return
 
-def generateGrid():
-    # generate empty tile grid prepopulated with placeholder characters
-    return
-
 def timer():
     # handles logic for timer
     return
 
-def randomizer():
-    # generate random list of letters to use as game tiles
-    return
+# generates random list of letters to use in game
+# allow letter repetition by default
+def letter_draw(count, letterSet, repeat=True):
+    drawnLetters = ()
+    while len(drawnLetters) < count:
+        index = random.randint(0, len(letterSet)-1)
+        if (not repeat and letterSet[index] in drawnLetters):
+            # choose another letter if already exists
+            continue
+        else:
+            # letters are allowed to repeat
```

Comment:
#letters are allowed to repeat or insert first instance of letter

Reply:
Comment updated